notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -181,7 +183,8 @@
     "if not os.path.exists(train_file):\n",
     "    download_and_extract(reviews_name, reviews_file)\n",
     "    download_and_extract(meta_name, meta_file)\n",
-    "    data_preprocessing(*input_files, sample_rate=sample_rate, valid_num_ngs=valid_num_ngs, test_num_ngs=test_num_ngs)"
+    "    data_preprocessing(*input_files, sample_rate=sample_rate, valid_num_ngs=valid_num_ngs, test_num_ngs=test_num_ngs)\n",
+    "    # data_preprocessing(*input_files, sample_rate=sample_rate, valid_num_ngs=valid_num_ngs, test_num_ngs=test_num_ngs, is_history_expanding=False)\n"
```

Comment:
add a comment to places like here:
# uncomment this like for the NextItNet model, because it does not need to unfold the user history

Reply:
also add the \#input_creator = NextItNetIterator  indicator

Reply:
added